platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/model/machine/Command.java

```diff
@@ -37,4 +37,9 @@
      * Returns command type (i.e. 'maven')
      */
     String getType();
+
+    /**
+     * Returns command preview URL. (i.e. 'http://awesome.com:8080/context')
+     */
+    String getPreviewUrl();
```

Comment:
pls, double check with this changes in API model personally i don't like it. 
But we can accept it if necessary for 4.0-beta 
@skabashnyuk @garagatyi @gazarenkov  your opinion need here